pkg/controller/clone-controller_test.go

```diff
@@ -332,6 +332,14 @@ var _ = Describe("Clone controller reconcile loop", func() {
 			AnnCloneRequest: "default/source", AnnPodReady: "true", AnnCloneToken: "foobaz", AnnUploadClientName: "uploadclient", AnnCloneSourcePod: "default-testPvc1-source-pod", AnnPodPhase: string(corev1.PodSucceeded)}
 		err = reconciler.client.Update(context.TODO(), testPvc)
 		Expect(err).ToNot(HaveOccurred())
+
+		sourcePod, err = reconciler.findCloneSourcePod(testPvc)
+		Expect(err).ToNot(HaveOccurred())
+		sourcePod.Status.Phase = corev1.PodSucceeded
```

Comment:
should't it already be corev1.PodSucceeded?

Reply:
Maybe it should. I've added this line because it wasn't.